### PR TITLE
XCOMMONS-2507: Add a priority concept in components

### PR DESCRIPTION
**Pull request by tmortagne**
* add priority support to wiki components

Depends on https://github.com/xwiki/xwiki-commons/pull/265.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -111,7 +111,7 @@
     <pgsql.version>42.6.0</pgsql.version>
     <oracle.groupId>com.oracle.database.jdbc</oracle.groupId>
     <oracle.artifactId>ojdbc8</oracle.artifactId>
-    <oracle.version>21.9.0.0</oracle.version>
+    <oracle.version>23.2.0.0</oracle.version>
```
Comment (surli) [inline pom.xml:114]: Shouldn't be part of this PR I think

Comment (tmortagne) [reply pom.xml:114]: Yes, I was in the wrong branch. It's in master too.
